lib/mocha/interface.js

```
@@ -242,14 +242,17 @@ function wrapPromisesForFiber(obj, methodList) {
       var f = new Future();
 
       var promise = original.apply(obj, arguments);
-      promiseAsThunk(promise)(function(error, value) {
-        if (error) {
-          f.throw(error);
-        } else {
-          f.return(value);
-        }
-      });
-      return f.wait();
+      // if only helper return a promise
+      if(promise) {
+        promiseAsThunk(promise)(function(error, value) {
+          if (error) {
+            f.throw(error);
+          } else {
+            f.return(value);
+          }
+        });
+        return f.wait();
+      }
```

@pahans Can we throw an error otherwise? (see my other comments to this PR)